plugin.xml

```diff
@@ -17,6 +17,11 @@
 
     <!-- Android -->
     <platform name="android">
+
+        <config-file target="AndroidManifest.xml" parent="/manifest">
```

Comment:
The parent attr shouldn't be "/" instead of "/manifest" ?

Reply:
Thank you for you input, I'm going to review this line!